docs/data/whats-new/WhatsNewLayout.tsx

```diff
@@ -0,0 +1,261 @@
+      'Premium plan announcement, including the most advanced features for data analysis and management.',
+    announcementDate: 'Thursday, May 12, 2022',
+    url: 'https://mui.com/blog/premium-plan-release/',
+    highlightList: [
+      { title: 'Row Grouping', url: '/x/react-data-grid/row-grouping/' },
+      { title: 'Excel export', url: '/x/react-data-grid/export/#exported-rows' },
+    ],
+  },
+  {
+    title: 'MUI X v5.0.0',
+    description: 'A new virtualization engine, and improvements in several APIs.',
+    announcementDate: 'Monday, Nov 22, 2021',
+    url: 'https://mui.com/blog/mui-x-v5/',
+    highlightList: [
+      {
+        title: 'New virtualization engine',
+        url: 'https://mui.com/blog/mui-x-v5/#a-new-virtualization-engine',
+      },
+      {
+        title: 'Improved state management',
+        url: 'https://mui.com/blog/mui-x-v5/#improved-state-management',
+      },
+      {
+        title: 'Simplified style customization',
+        url: 'https://mui.com/blog/mui-x-v5/#simplified-style-customization',
+      },
+    ],
+  },
+];
+
+function BlogCard(props: { blog: Blog }) {
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        px: 2,
+        pt: 2,
+        pb: 0,
+        gap: 1.5,
+        borderRadius: '12px',
+        backgroundColor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? 'rgba(0, 27, 55, 0.2)'
+            : `${alpha(theme.palette.grey[50], 0.4)}`,
+        borderColor: 'divider',
+      }}
+      component="article"
+      variant="outlined"
+    >
+      <CardContent sx={{ p: 0 }}>
+        <Box
+          sx={{
+            borderBottom: '1px solid',
+            borderColor: 'divider',
+            marginBottom: 1,
+            px: 1,
+          }}
+        >
+          <Typography variant="caption" color="text.secondary">
+            {props.blog.announcementDate}
+          </Typography>
+          <Typography gutterBottom variant="h6" component="div">
+            {props.blog.title}
+          </Typography>
+        </Box>
+
+        <Box sx={{ p: 1 }}>
+          <Typography variant="body2" component="div" color="text.secondary">
+            {props.blog.description}
+          </Typography>
+
+          <List sx={{ pb: 0 }}>
+            {props.blog.highlightList.map((item) => (
+              <ListItem
+                key={item.title}
+                sx={{
+                  py: 0.5,
+                  px: 0.5,
+                  display: 'flex',
+                  alignItems: 'flex-start',
+                  lineHeight: '20px',
+                  [`&:before`]: {
+                    content: '"•"',
+                    color: 'grey',
+                    lineHeight: '20px',
+                  },
+                }}
+              >
+                <Typography sx={{ pl: 1 }} variant="body2">
+                  <Link href={item.url}>{item.title}</Link>
+                </Typography>
```

Comment:
WDYT about making links look like links instead of bolded list items? 🤔 
https://mui.com/material-ui/react-link/#accessibility
```suggestion
                <Link href={item.url} sx={{ pl: 1 }} variant="body2">{item.title}</Link>>
```

Reply:
I think the list aspect is meaningful. Regarding the underline, I was using it before, but based on the branding provider, however, links don't have it. (It does make the page visually better, I think).
Regarding the bold, you're right, it's a tad too much in this context. I made it a little lighter.